test-requirements.txt

```diff
@@ -1,3 +1,4 @@
+argon2-cffi>=16.1.0; python_version >= '3'
```

Comment:
@jeromelebleu I don't remember why this dependency is only installed with python3?

Reply:
I don't remember too as of now... As I didn't find any reason after a quick search - it is said that [argon2-cffi](https://pypi.org/project/argon2-cffi/) is tested with Python 2 - I suggest to remove the requirement on the Python version.

Reply:
Fine with me. Can you do it please?

Reply:
It's done!